client/blocks/props-viewer/style.scss

```diff
@@ -0,0 +1,39 @@
+.props-viewer__card {
+  color: darken($gray, 10%);
+}
+
+.props-viewer__usage {
+  margin: 28px -25px -17px;
+}
+
+.props-viewer__card p {
```

Comment:
Since the targeted `p` already has a `className`, we should use that: `.props-viewer__description` (in general good to avoid element selectors in styles, unless the cascade is very intentional)